src/routes/article.py

```
@@ -24,39 +24,57 @@
+        elif body.board_id == 6:  # grant
+            await send_discord_bot_request_no_reply(action_code=1002, body={'channel_id': get_settings().grant_channel_id, 'content': body.content})
+    except Exception: logger.error(f'err_type=create_article ; error occurred during connecting to discord ; {body=}', exc_info=True)
     return ret
 
 
 # This works as "api/article" + "s/{board_id}" (="api/articles/{board_id}")
 @article_general_router.get('s/{board_id}')
 async def get_article_list_by_board(board_id: int, session: SessionDep, request: Request) -> list[ArticleResponse]:
-    user = get_user(request)
     board = session.get(Board, board_id)
     if not board: raise HTTPException(404, detail="Board not found")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)